client.go

```diff
@@ -145,15 +146,26 @@ func New(end_point, accessid, accesskey, instance_name string, kwargs ...interfa
 		return nil, (OTSClientError{}.Set("host of end_point should be specified, e.g. http://ots.aliyuncs.com:80."))
 	}
 
+	// add http transport
+	if o.MaxConnection == 0 {
+		o.MaxConnection = 2000
+	}
+	trans := &http.Transport{
+		MaxIdleConnsPerHost: o.MaxConnection,
+		Dial: (&net.Dialer{
+			Timeout: time.Duration(15) * time.Second,
+		}).Dial,
+	}
+
 	// set default setting for urllib
 	url_setting := urllib.HttpSettings{
 		false,            // ShowDebug
 		"GiterLab",       // UserAgent
-		60 * time.Second, // ConnectTimeout
-		60 * time.Second, // ReadWriteTimeout
+		15 * time.Second, // ConnectTimeout
```

Comment:
Is this change made by you or pulled from somewhere?

Reply:
I rebased again, only mine remains.